.github/workflows/continuous-integration-workflow.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        node: [ '18' ]
+        node: [ '18.15.x' ]
```

Comment:
18.15.x  doesn't look like the is to install a latest version of Node.js 18 - do we have any limitations on Node.js version related to VSCode OS Tools and/or test?